### PR TITLE
Readme sync

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: .github/workflows/doc.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  push:
+    branches:
+      - main
+      - readme-sync
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: readmeio/github-readme-sync@v2
```
Comment (denisok) [inline .github/workflows/doc.yml:12]: @atymchuk could you please clone this action? others should be there already.

File: .github/workflows/doc.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        with:
+          readme-oas-key: ${{ secrets.README_TOKEN }}
```
Comment (denisok) [inline .github/workflows/doc.yml:14]: @atymchuk could you please check if this token is correct.

Comment (atymchuk) [reply .github/workflows/doc.yml:14]: Not really - the tokens are only known at the time they are posted. If @BupycHuk grabbed it from the readme.io's site it should be OK. This run indirectly confirms it's OK, so should be good to go.

File: .github/workflows/doc.yml (hunk 1 above)
Comment (denisok) [reply .github/workflows/doc.yml:12]: @BupycHuk please change all actions to the percona-platform.

**Review by denisok — CHANGES_REQUESTED** [review]

File: .github/workflows/doc.yml (hunk 1 above)
Comment (atymchuk) [reply .github/workflows/doc.yml:12]: cloned!

**Review by atymchuk — DISMISSED** [review]

**Comment by atymchuk** [comment]
@denisok all actioned are sourced from /percona-paltform now )

**Review by denisok — APPROVED** [review]

**Review by denisok — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]
:shipit:

**Pull request merged**